src/component/TodoList/TodoList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Todo } from '../Todo/Todo';
+import './TodoList.scss';
+
+export const TodoList = ({ todos }) => (
+  <ul className="todo__list">
+    {todos.map(item => (
+      <li className="todo__item" key={item.id}>
+        <Todo
+          title={item.title}
+          user={item.user}
+          completed={item.completed}
+        />
```

Comment:
your version is also ok, but it's better to use spread operator
https://kevinyckim33.medium.com/jsx-spread-operator-component-props-meaning-3c9bcadd2493

```suggestion
        <Todo {...item}/>
```